src/drivers/net/ena.c

```diff
@@ -350,6 +350,89 @@ static int ena_admin ( struct ena_nic *ena, union ena_aq_req *req,
 	return rc;
 }
 
+/**
+ * Set async event notification queue config
+ *
+ * @v ena		ENA device
+ * @ret rc		Return status code
+ */
+static int ena_set_aenq_config ( struct ena_nic *ena,
+				 uint32_t enabled_groups ) {
+	union ena_aq_req *req;
+	union ena_acq_rsp *rsp;
+	union ena_feature *feature;
+	int rc = 0;
```

Comment:
Please do not initialise `rc = 0`.  Initialising variables unnecessarily prevents the compiler from detecting situations in which a code path has genuinely forgotten to set a variable's value.